web/src/i18n/locales/en.yaml

```diff
@@ -282,39 +282,39 @@
       modularity: 
         header: "Modularity"
         subheader: "This condition requires the use of a modular, flexible approach to systems development, including the use of open interfaces and exposed application programming interfaces (API); the separation of business rules from core programming; and the availability of business rules in both human and machine-readable formats. The commitment to formal system development methodology and open, reusable system architecture is extremely important in order to ensure that states can more easily change and maintain systems, as well as integrate and interoperate with a clinical and administrative ecosystem designed to deliver person-centric services and benefits. Modularity is breaking down systems requirements into component parts. Extremely complex systems can be developed as part of a service-oriented architecture (SOA). Modularity also helps address the challenges of customization. Baseline web services and capabilities can be developed for and used by anyone, with exceptions for specific business processes handled by a separate module that interoperates with the baseline modules. With modularity, changes can be made independently to the baseline capabilities without affecting how the extension works. By doing so, the design ensures that future iterations of software can be deployed without breaking custom functionality. A critical element of compliance with this condition is providing CMS with an understanding of where services and code will be tightly coupled, and where the state will pursue a more aggressive decoupling strategy."
-        title: "Describe how this activity will support the Modularity condition"
+        title: "Describe how this activity will support the modularity condition"
```

Comment:
Since these are proper titles/names of the standards/condition, I'm OK with them staying capitalized (apply this for all list conditions below this. What do you think?

Reply:
Yeah, I wasn't sure if it was a proper name or just the name. I agree they should stay capitalized for these instances.